_posts/2016-06-20-recent-graduate-at-skybet.md

```diff
@@ -0,0 +1,56 @@
+---
+layout:     post
+title:      A Recent Graduates Guide to Sky Betting And Gaming…
+date:       2016-06-20 10:00
+summary:    A recent graduates blog on their first few months at Sky Betting and Gaming as well as what to expect.
+category:   Graduates, Data Engineering
+tags:       Leeds, Data, Engineer, Graudate
+author:     marcus_ojerinde_ardalla
```

Comment:
You'll need to add this author tag to `_data/authors.yml` to display author information for this post.